shell/platform/darwin/ios/rendering_api_selection.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_IOS_RENDERING_API_SELECTION_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_IOS_RENDERING_API_SELECTION_H_
+
+#include <objc/runtime.h>
+
+#include "flutter/fml/macros.h"
+
+namespace flutter {
+
+enum class IOSRenderingAPI {
+  kSoftware,
+  kOpenGLES,
+  kMetal,
+};
+
+IOSRenderingAPI GetRenderingAPIForProcess();
+
+Class GetQuartzLayerClassForRenderingAPI(
```

Comment:
Man, `Quartz` sounds so 2005, is that really how it is still referred to in the API?

Reply:
No. Thats just me dating myself for no reason. Changed to `GetCoreAnimationLayerClassForRenderingAPI()`.